Buildings/Fluid/HeatExchangers/BaseClasses/PartialEffectivenessNTU.mo

```diff
@@ -103,8 +103,8 @@ initial equation
   Q_flow_nominal = m1_flow_nominal*cp1_nominal*(T_a1_nominal - T_b1_nominal);
   if use_Q_flow_nominal then
     Q_flow_nominal = -m2_flow_nominal*cp2_nominal*(T_a2_nominal - T_b2_nominal);
-    eps_nominal = abs(Q_flow_nominal/((T_a1_nominal - T_a2_nominal)*
-      CMin_flow_nominal));
+    eps_nominal = Q_flow_nominal/((T_a1_nominal - T_a2_nominal)*
```

Comment:
This file is from https://github.com/ibpsa/modelica-ibpsa
If you need changes, it needs to be fixed on the IBPSA library, please see https://github.com/ibpsa/modelica-ibpsa/wiki/Workflow-for-code-changes
This is then merged to Buildings using https://simulationresearch.lbl.gov/modelica/buildingspy/development.html#buildingspy.development.merger.IBPSA.merge